### PR TITLE
Added documentation about db migration

### PR DESCRIPTION
**Pull request by rebert**
Added some tipps about how to handle the db migration

### CODE REVIEW

**Review by boecklic — CHANGES_REQUESTED** [review]
Looks good. Maybe you can also include a section that describes the process to 'clean' migrations for a PR (i.e. migrate back, delete migration files, create one new one, migrate forward). 
You might also make a note about backwards migration (in which cases it work automatically, in which cases one should pay attention (e.g. if fields are deleted)).

File: README.md [hunk 1]
```diff
@@ -149,6 +149,40 @@ DISABLE_LOGGING=1 ./manage.py shell
...
+**How to put the database to the state of a previous code base?**
+
+With the following command of the Django shell a specific state of the database can be achieved:
+```bash
+.manage.py migrate stac_api 0003_auto_20201022_1346
+```
+
+**How to get a working database when migrations scripts screw up?**
+
+With the following commands it is possible to get a proper state of the database:
```
Comment (procrastinatio) [inline README.md:180]: Add a warning, `reset_db` is pretty much desctuctive :bomb:

**Review by procrastinatio — APPROVED** [review]
OK.

File: README.md (hunk 1 above)
Comment (ltshb) [reply README.md:180]: Agreed on this

**Review by ltshb — COMMENTED** [review]
Good description :+1:, same comment as Chris about the PR migration merge.

**Comment by rebert** [comment]
@boecklic

**Review by ltshb — APPROVED** [review]
:+1:

**Pull request merged**